### PR TITLE
TST: Note save and persist e2e test

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/5renvuyvrdpw6flplmbzt2vfvq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1831)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/avmh5pwwdkweyo6xmhlsswiaxu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1832)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ljypztwgecituxfjdbwpm4a3za?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1834)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/djeqjgkd6l7ervockkvrd47zjq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1837)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/kelrfgzvx3f5xkhkirfdncofp4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1847)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/74hcwuu2basehe7gg762ujzoke?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1852)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/33havkxxty6ea5nmjs5v6omaky?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1854)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/54at6vox5kzl4rdldvh6bavzxi?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1855)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: tests/e2e/bluewallet2.spec.js [hunk 1]
```diff
@@ -469,11 +469,10 @@ describe('BlueWallet UI Tests - import BIP84 wallet', () => {
...
 
-    // back to wallet screen
-    await device.pressBack();
-    await device.pressBack();
-
-    // open CoinControl
+    // Terminate and reopen the app to confirm the note is persisted
+    await device.launchApp({ newInstance: true });
+    await yo('WalletsList');
+    await element(by.text('Imported HD SegWit (BIP84 Bech32 Native)')).tap();
```
Comment (Overtorment) [inline tests/e2e/bluewallet2.spec.js:475]: noice. i forgot we had a test for the tx note. so we needed only to add app restart

**Review by Overtorment — APPROVED** [review]
lfg

**Pull request merged**